x-pack/platform/plugins/private/translations/translations/fr-FR.json

```diff
@@ -7261,6 +7260,14 @@
     "securitySolutionPackages.alertSuppressionRuleDetails.upsell": "La suppression d'alertes est configurée mais elle ne sera pas appliquée en raison d'une licence insuffisante",
     "securitySolutionPackages.alertSuppressionRuleForm.upsell": "La suppression d'alertes est activée avec la licence {requiredLicense} ou supérieure",
     "securitySolutionPackages.beta.label": "Bêta",
+    "securitySolutionPackages.csp.cspEvaluationBadge.failLabel": "Échec",
```

Comment:
Why did the order of all of these translations change?

Reply:
I don't know. Just the output of running i18n script. It looks like they just moved in the file and are not new.